README.md

```diff
@@ -97,6 +97,33 @@ open http://myapp.lvh.me
 
 And enjoy previewing your app!
 
+#### Anatomy of a Kaiserfile
+
+A Kaiserfile is made up of statements, which are different commands followed by some parameters. Each command is just a ruby method so you can call it that way. Only the `dockerfile` command is required. All the other commands are either not required or have defaults.
+
+- `plugin` - Takes 1 parameter: the name of the plugin you wish to use in this Kaiserfile.
```

Comment:
An example for each would be great way to easily demonstrate how the commands are intended to be used, but not blocking for this PR 😄